### PR TITLE
Update jenkinsfile

### PR DESCRIPTION
**Pull request by gmenih341**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,77 +1,6 @@
...
-            currentBuild.result = 'NOT_BUILT'
-            throw err
-        }
-
-        node {
-            deploy(projectBaseName, imageName, "production", "deploy", "gcs-repo/erento-app")
-            milestone(label: "deployed to production")
-        }
-    }
+toolPipeline {
```
Comment (rokerkony) [inline Jenkinsfile]: you should not use tool pipeline but new pipeline for nestJs service

File: package.json [hunk 2]
```diff
@@ -15,8 +15,7 @@
     "lint": "tslint -c tslint.json -e \"**/node_modules/**\" -p ./tsconfig.json",
     "lint:fix": "tslint -c tslint.json --fix -e \"**/node_modules/**\" -p ./tsconfig.json",
     "lint-staged": "lint-staged",
-    "postbuild": "npm prune --production",
-    "prebuild": "npm run clean-dist",
```
Comment (rokerkony) [inline package.json]: this should be still running

File: package.json [hunk 3]
```diff
@@ -15,7 +15,7 @@
     "lint": "tslint -c tslint.json -e \"**/node_modules/**\" -p ./tsconfig.json",
     "lint:fix": "tslint -c tslint.json --fix -e \"**/node_modules/**\" -p ./tsconfig.json",
     "lint-staged": "lint-staged",
-    "postbuild": "npm prune --production",
+    "clean-pre-prod": "npm prune --production",
```
Comment (rokerkony) [inline package.json]: why do we need this? postbuild runs automatically

Comment (gmenih341) [reply package.json]: I found it in the `toolPipeline`, and thought it's a good idea. It makes it a bit easier to work with the repo, so you can build it locally without deleting `devDelendencies`.

Comment (rokerkony) [reply package.json]: alright :)

**Pull request merged**